packages/ra-core/src/core/useConfigureAdminRouterFromChildren.tsx

```diff
@@ -10,8 +10,9 @@ import {
     useEffect,
     useState,
 } from 'react';
-import { useLogout, usePermissions } from '../auth';
-import { useSafeSetState } from '../util';
+import useLogout from '../auth/useLogout';
+import usePermissions from '../auth/usePermissions';
+import { useSafeSetState } from '../util/hooks';
```

Comment:
I don't understand the rationale of this change.

Reply:
Avoiding importing internal stuff from barrel files should improve our tests performances. I'm changing it when I see some